test/unit/tcache_max.c

```diff
@@ -2,6 +2,8 @@
 #include "test/san.h"
 
 const char *malloc_conf = TEST_SAN_UAF_ALIGN_DISABLE;
+extern void tcache_bin_info_compute(
+    cache_bin_info_t tcache_bin_info[TCACHE_NBINS_MAX]);
```

Comment:
Probably cleaner to still put in the _extern header this way:
```
JET_EXTERN(void tcache_bin_info_compute(
    cache_bin_info_t tcache_bin_info[TCACHE_NBINS_MAX]))
```

and define JET_EXTERN(func) as ```extern func;``` under the test build

Reply:
If put in `tcache_externs.h`, the function is already exposed and there is no need to add `extern` for test to use it. I though we want as few functions in `tcache_externs.h` as possible?

Reply:
I was thinking to define it to null for test. Never mind the current way looks fine too. Let's leave it as is.